get-es-instance-type.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Returns the elasticsearch instance count defined in each environment
+
+# Usage
+#   ./get-keys.sh develop
+
+# Arguments
+#   - $1 - the branch to check
+
+[ -z "$1" ] && echo "The branch name to check must be provided as the \$1 argument." && exit 1
+
+BRANCH=$1
+
+if [[ $BRANCH == 'develop' ]] ; then
+  echo "t2.small.elasticsearch"
+elif [[ $BRANCH == 'experimental1' ]] ; then
+  echo "t2.small.elasticsearch"
+elif [[ $BRANCH == 'experimental2' ]] ; then
+  echo "t2.small.elasticsearch"
+elif [[ $BRANCH == 'experimental3' ]] ; then
+  echo "t2.small.elasticsearch"
+elif [[ $BRANCH == 'irs' ]] ; then
+  echo "t2.small.elasticsearch"
+elif [[ $BRANCH == 'staging' ]] ; then
+  echo "t2.medium.elasticsearch"
+elif [[ $BRANCH == 'test' ]] ; then
+  echo "t2.medium.elasticsearch"
```

Comment:
@mmarcotte note that this script has been added to define the Elasticsearch instance size per environment. Right now, 'test' is medium and 'migration' is small. Beef it up right now for test migration runs? Or leave it as is?

Reply:
S'ok for now. We will make adjustments as needed and informed by https://github.com/flexion/ef-cms/issues/6654. Thanks @julialeague. Still consider this a work in progress, and that is👌.